reducing_utils.go

```diff
@@ -44,10 +47,24 @@ References:
 	http://www.oasis-open.org/committees/documents.php?wg_abbrev=office-formula
 	OpenDocument-formula-20090508.odt
 */
-func Pmt(rate float64, nper int64, pv float64, fv float64, when paymentperiod.Type) float64 {
-	factor := math.Pow(1.0+float64(rate), float64(nper))
-	secondFactor := (factor - 1) * (1 + rate*when.Value()) / rate
-	return -(pv*factor + fv) / secondFactor
+func Pmt(rate decimal.Decimal, nper int64, pv int64, fv int64, when paymentperiod.Type) decimal.Decimal {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
this func shouldn't panicked. which line did?

Reply:
yes, it shouldn't panic now. I added it initially to handle nan case when I was gradually changing float to decimal. I forgot to remove it.